maps/RadioTower2F.asm

```diff
@@ -100,8 +100,6 @@ TrainerGruntF2:
 Buena:
 	faceplayer
 	opentext
-	checkflag ENGINE_ROCKETS_IN_RADIO_TOWER
-	iftrue .MidRocketTakeover
```

Comment:
For now I suggest keeping these in case its decided that this change should be locked behind a flag. I suspect that it probably won't need to be though, so once this gets properly merged into the main branch for speedchoice V7 we should propose to delete this.